data/CBID2.json

```diff
@@ -35,3 +35,6 @@
     }
   ]
 }
+"testimonials": [
```

Comment:
This section needs to go inside the `}` between line `36` and `37`

Reply:
Ok

Reply:
> This section needs to go inside the `}` between line `36` and `37`

I think I got it

Reply:
Now you just need a `,` on line `36`

Reply:
Done